testng.xml

```diff
@@ -1,15 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Kindly see the feedbacks given in the last sessions and follow them
Create a testngnationaltiy.xml, do not modify the existing testng.xml given in the project